### ISSUE TITLE
Cannot get FIRE type blocks

### ISSUE THREAD

**Issue by YouHaveTrouble** [LoneDev6/API-ItemsAdder#1]
https://paste.gg/p/anonymous/1363319f2504473eb8e3e924483fe34e

The line stacktrace points is
```java
CustomBlock block = CustomBlock.getInstance("customfire:green_fire");
```

Not sure if that's important, but I'm depending on the same jar file I use on the server I test this on, so it's not due to different obfuscation.

**Comment by LoneDev6** [comment]
Moved https://github.com/PluginBugs/Issues-ItemsAdder/issues/1421

**Comment by LoneDev6** [comment]
Implemented in 3.1.1
